### PR TITLE
Move Ubuntu to separate folder

### PR DESCRIPTION
**Pull request by it-mak**

### CODE REVIEW

File: ubuntu/README.md [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+  [2]: https://www.azul.com/
+  [3]: https://www.azul.com/products/core/
+  [4]: https://hub.docker.com/r/azul/zulu-openjdk-alpine
+  [5]: https://hub.docker.com/r/azul/zulu-openjdk-centos
+  [6]: https://hub.docker.com/r/azul/zulu-openjdk-debian
+  [7]: https://hub.docker.com/r/azul/zulu-openjdk
+  [8]: https://hub.docker.com/r/azul/zulu-openjdk/tags?name=distroless
+
+
+  [19]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/19-jre-headless-latest/Dockerfile
```
Comment (AntonKozlov) [inline ubuntu/README.md:371]: This should point to https://github.com/zulu-openjdk/zulu-openjdk/

File: ubuntu/README.md [hunk 2]
```diff
@@ -0,0 +1,673 @@
...
+  [202]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u282-8.52.0.23/Dockerfile
+  [203]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u302-8.56.0.21/Dockerfile
+  [204]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u312-8.58.0.13/Dockerfile
+  [205]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u322-8.60.0.21/Dockerfile
+  [206]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u332-8.62.0.19/Dockerfile
+  [207]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u342-8.64.0.15/Dockerfile
+  [208]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u345-8.64.0.19/Dockerfile
+  [209]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u352-8.66.0.15/Dockerfile
+  [210]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.19/Dockerfile
+  [211]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.21/Dockerfile
```
Comment (AntonKozlov) [inline ubuntu/README.md:612]: File not found, https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.21/ -- docker files had platform suffix at that moment (e.g. Dockerfile_x86)

File: ubuntu/README.md (hunk 1 above)
Comment (it-mak) [reply ubuntu/README.md]: Fixed

File: ubuntu/README.md (hunk 2 above)
Comment (it-mak) [reply ubuntu/README.md]: All Dockerfile_x86 Dockerfile_arm were combined to one Dockerfile as it was at the start.

**Comment by AntonKozlov** [comment]
LGTM, thanks!

**Pull request merged**